pkg/controller/nodes/watchers.go

```diff
@@ -33,6 +33,34 @@ func (r *ReconcileNodes) watchDeletions(stop <-chan struct{}) (chan string, erro
 	return chDels, nil
 }
 
+func (r *ReconcileNodes) watchUpdates() (chan string, error) {
```

Comment:
One option is to use the same informer as watchDeletions, and just return two channels, keeping a single connection. Don't have an opinion on that however.